### PR TITLE
Auto create first Schedule

### PR DESCRIPTION
**Pull request by ezequielc**

### CODE REVIEW

File: web/scripts/editor/services/svc-editor-factory.js [hunk 1]
```diff
@@ -156,8 +156,22 @@ angular.module('risevision.editor.services')
 
               $state.go('apps.editor.workspace.artboard', {
                 presentationId: resp.item.id
+              }).then(function () {
```
Comment (ezequielc) [inline web/scripts/editor/services/svc-editor-factory.js]: I added it after $state.go() as we close modals when changing state.

**Comment by ezequielc** [comment]
@alex-deaconu Please review. Thanks

**Comment by ezequielc** [comment]
@alex-deaconu You can hold on the review, I found an issue when clicking Save and Preview. I'll let you know when I fix it. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc sounds good. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu I added a fix for the Save and Preview issue. It is not an elegant solution but we can revisit it later. Please review. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc reviewed, looks good 👍 .  Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu Sorry, I missed your comment, I already squashed.
But as it breaks the functionality of auto creating the Schedule, I think it should be deployed with this PR.
Thoughts? Thanks

**Comment by alex-deaconu** [comment]
@ezequielc no worries. We should be good. Thanks.

**Pull request merged**